src/components/Catalog/CategoryName/CategoryName.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/utils/vars.scss';
+@import '../../../styles/utils/mixins.scss';
+
+.categoryName {
+  width: 100%;
+  margin: 40px 0;
+
+  @media (max-width: 320px) {
+    margin: 32px 0;
+  }
+
+  &-text {
+    font-family: $global-font;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 56px;
+    color: $primary-color;
+
+    padding-bottom: 8px;
+    margin: 0;
+
+    @media (max-width: 320px) {
+      font-size: 32px;
+    }
+  }
+
+  &-quantity {
+    font-family: $global-font;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 21px;
+    color: $secondary-color;
```

Comment:
same here and in further cases, use prepared solutions from _mixins and _vars in every appropriate case